test/smoke/run-smoke-suite.sh

```diff
@@ -132,6 +132,7 @@ spec:
     - {name: ALLURE_PROJECT_ID, value: "${ALLURE_PROJECT_ID:-}"}
     - {name: ALLURE_TOKEN, value: "${ALLURE_TOKEN:-}"}
     - {name: ALLURE_RESULTS, value: "${ALLURE_RESULTS:-/tests/_out/allure_report}"}
+    - {name: TEST_RC, value: "${TEST_RC:-'True'}"}
```

Comment:
@denisqsound are we sure we want to set `true` by default here?

Reply:
What value do we want to have by default here?

Reply:
I've set it to false